xml/ha_storage_protection.xml

```diff
@@ -809,17 +809,50 @@ Timeout (msgwait)  : 10
      </callout>
    </calloutlist>
   </step>
-  <step>
-   <para>For a two-node cluster, decide if you want predictable or random delays. For
-   other cluster setups you do not need to set this parameter.</para>
+  <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>
+    For a two-node cluster, in case of split-brain, there will be fencing issued from
+    each node to the other as expected. To prevent both nodes from practically
+    being reset at the same time, it is recommended to apply the following fencing
+    delays to help one of the nodes, or even the preferred node, win the fencing match.
+    For clusters with more than two nodes, you do not need to apply these delays.
+   </para>
    <variablelist>
+    <varlistentry>
+     <term>Priority fencing delay</term>
+     <listitem>
+       <para>
+        By configuring the <literal>priority-fencing-delay</literal> cluster property
+        with a delay value (by default disabled), if the other node is lost and it has
+        the higher total resource priority, the fencing targeting it will be delayed for
+        the specified amount of time. So that the more important node wins fencing
+        match in case of split-brain.
```

Comment:
```suggestion
        The <literal>priority-fencing-delay</literal> cluster property is disabled by
        default. By configuring a delay value, if the other node is lost and it has
        the higher total resource priority, the fencing targeting it will be delayed
        for the specified amount of time. This means that in case of split-brain,
        the more important node wins the fencing match .
```

Reply:
I have split 2 sentences into 3 to try to make their construction simpler. This removes a parenthetical phrase, and in the last sentence, it puts it in the order [if condition] [outcome], which I think is clearer.